frontend/public/components/pod.tsx

```diff
@@ -651,9 +654,53 @@ export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => {
     (status === 'CrashLoopBackOff' || status === 'ErrImagePull' || status === 'ImagePullBackOff') &&
     containerStatusStateWaiting
   ) {
+    let footerLinks: React.ReactNode;
+    let headerTitle = '';
+    if (status === 'CrashLoopBackOff') {
+      headerTitle = t('public~Pod crash loop back-off');
+      const containers: ContainerSpec[] = pod.spec?.containers ?? [];
+      footerLinks = (
+        <div>
+          {t(
+            'public~CrashLoopBackOff indicates that the application within the container is failing to start properly.',
+          )}
+          <div className="co-divider-padding">
```

Comment:
If you use a `p` element instead, can you remove `co-divider-padding`?

Reply:
With <p> there is no space:
![Screenshot 2021-09-02 at 20-25-11 Pods · OKD](https://user-images.githubusercontent.com/18728857/131941220-94b711c3-900e-4e8f-ac8e-37b27643ad4a.png)

Reply:
Got this working with the `p` element